single_group_playbooks/slurm.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: Import pre deploy checks.
+  ansible.builtin.import_playbook: pre_deploy_checks.yml
+
+- name: Configure slurm for clients and management machines
+  hosts:
+    - compute_vm
+    - sys_admin_interface
+    - user_interface
+  roles:
+    - slurm
+...
```

Comment:
'slurm' is currently not an inventory group. How is this play different from `single_role_playbooks/slurm.yml`?

Reply:
It's not.

I guess I misunderstood that we have the `single_role_playbooks` for the roles. And that we have `single_group_playbooks` for **groups** of playbooks (like `slurm` would be `slurm_client` and `slurm_managment`, and the same for `rsyslog` for `rsyslog_client` ...). If this is indeed only for **inventory** groups, then "whoops", I will remove it.

Reply:
Single group playbooks was indeed intended for inventory groups.